src/main/java/frc/robot/util/MiscUtils.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.util;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.Constants;
+
+public class MiscUtils {
+  /**
+   * @param point The Pose2d that needs to be reflected across the midline
+   * @return The reflected Pose2d (Midline is at x = 8.27 field-based)
+   */
+  public static Pose2d reflectAcrossMidline(Pose2d point) {
+    return new Pose2d(
+        point.getX() + (2.0 * (point.getX() - Constants.Landmarks.CENTER_LINE_LOCATION)),
+        point.getY(),
+        new Rotation2d(
+            point.getRotation().getDegrees() + (2.0 * (90.0 - point.getRotation().getDegrees()))));
```

Comment:
should be radians, `π - point.getRotation().getRadians()` simplified

Reply:
done